crates/fluvio-cli/src/consume/mod.rs

```diff
@@ -644,6 +652,16 @@ impl ConsumeOpt {
                 )
                 .bold()
             );
+        // If --end-offset=X
+        } else if let Some(end) = self.end_offset {
+            eprintln!(
+                "{}",
+                format!(
+                    "Consuming records until record {} in topic '{}'",
```

Comment:
I think that in this case, the message should still mention that it will start from the end